scripts/github/setup/setup-github-project.sh

```
@@ -0,0 +1,714 @@
+- component: project-setup" \
+            --label "priority: medium,type: task,component: project-setup" \
+            --repo "$repo_owner/$repo_name")
+        
+        local issue_number=$(echo "$issue" | grep -o '[0-9]*$')
+        
+        if [ -n "$issue_number" ]; then
+            print_success "Created issue: #$issue_number - $issue_title"
+            
+            # Add to project if project ID exists
+            if [ -f ".github/project-id.txt" ]; then
+                local project_id=$(cat .github/project-id.txt)
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)